scripts/exgfs_wave_post_pnt.sh

```
@@ -72,68 +69,45 @@
   mkdir -p ${STA_DIR}/bull
   mkdir -p ${STA_DIR}/cbull
 
-  set +x
-  echo ' '
-  echo 'Grid information  :'
-  echo '-------------------'
-  echo "   Output points : $waveuoutpGRD"
-  echo ' '
-  set_trace
+  printf "\n   Grid information  :\n   ------------------\n     Output points : %s\n" "${waveuoutpGRD}"
 
 # --------------------------------------------------------------------------- #
 # 1.  Get files that are used by most child scripts
 
-  exit_code=0
-
-  set +x
-  echo ' '
-  echo 'Preparing input files :'
-  echo '-----------------------'
-  set_trace
+  printf "\nPreparing input files :\n-------------------------\n"
 
 # 1.a Model definition files and output files (set up using poe)
 
 # Copy model definition files
   iloop=0
   for grdID in ${waveuoutpGRD}; do
     if [[ -f "${COMIN_WAVE_PREP}/${WAV_MOD_TAG}.mod_def.${grdID}.bin" ]]; then
-      set +x
       echo " Mod def file for ${grdID} found in ${COMIN_WAVE_PREP}. copying ...."
-      set_trace
 
       cp -f "${COMIN_WAVE_PREP}/${WAV_MOD_TAG}.mod_def.${grdID}.bin" "mod_def.${grdID}"
```

```suggestion
      cpreq "${COMIN_WAVE_PREP}/${WAV_MOD_TAG}.mod_def.${grdID}.bin" "mod_def.${grdID}"
```

---

The `-f` threw me off here.  This forces an overwrite, but I wasn't sure if it would suppress errors for non-existent files.  Testing shows that it _does not_ suppress errors, and the `-f` is a valid argument to `cpreq`, so I will adapt it.